app/models/course/experience_points/forum_disbursement.rb

```diff
@@ -47,7 +47,9 @@ def end_time=(end_time_param)
   # This will be pro-rated based on the number of weeks in the period.
   #
   # @return [Integer]
-  attr_reader :weekly_cap
+  def weekly_cap
+    @weekly_cap ||= weekly_cap_default
```

Comment:
Not sure if we need to have a `weekly_cap_default` like that though, versus `@weekly_cap ||= 100`. Small issue anyway.